content/tutorials/rust/mysql-cdc.md

```diff
@@ -0,0 +1,199 @@
+are related. For this example, we're going to create a topic that will receive all of our
+MySQL events. Run the following command to create the topic:
+
+```bash
+$ fluvio topic create rust-mysql-cdc
+```
+
+## Start the Producer and Consumer
+
+Now we'll launch the CDC Producer, which will watch for any SQL commands that are executed
+in the leader MySQL instance and produce corresponding Fluvio events.
+
+```bash
+cdc-mysql$ cargo run --bin cdc-producer -- ./producer_profile.toml
+```
+
+In another terminal window, we'll launch the CDC Consumer, which listens for new Fluvio
+events and replicates them in the follower MySQL instance.
+
+```bash
+cdc-mysql$ cargo run --bin cdc-consumer -- ./consumer_profile.toml
+```
+
+## Connect to Mysql
+
+Now you're ready to start interacting with your databases. We'll open up
+the `mysql` command line and start running queries on the leader database, and we'll
+open a separate window connected to the follower database to see that all the changes
+get propagated.
+
+### Connect to Producer (leader DB)
+
+The leader database is bound to port 3080. You can connect to it using the `mysql`
+command as follows:
+
+```
+$ mysql -h 0.0.0.0 -P 3080 -ufluvio -pfluvio4cdc!
+...
+mysql >
+```
+
+The producer profile has a filter that registers only changes applied to the "flvDB"
+database. Let's create the database now:
+
+```
+mysql> CREATE DATABASE flvDb;
+Query OK, 1 row affected (0.01 sec)
+
+mysql> use flvDb;
+Database changed
+```
+
+### Connect to Consumer (follower DB)
+
+The follower database is bound to port 3090. Open another terminal window and connect
+to it using the `mysql` command:
+
+```
+$ mysql -h 0.0.0.0 -P 3090 -ufluvio -pfluvio4cdc!
+...
+mysql >
+```
+
+Since the CDC Consumer is already running, the "flvDb" database we just created on
+the leader will also be created on the follower!
+
+```
+mysql> SHOW DATABASES;
++--------------------+
+| Database           |
++--------------------+
+| flvDb              |
+| information_schema |
++--------------------+
+2 rows in set (0.00 sec)
+
+mysql> use flvDb;
+Database changed
+```
+
+## MYSQL Test Commands
+
+In the producer terminal, generate mysql commands and see then propagated to the consumer database;
+
+### Producer
+
+```
+mysql> CREATE TABLE pet (name VARCHAR(20), owner VARCHAR(20), species VARCHAR(20), sex CHAR(1), birth DATE);
+Query OK, 0 rows affected (0.03 sec)
+
+mysql> show tables;
++-----------------+
+| Tables_in_flbDb |
++-----------------+
+| pet             |
++-----------------+
+1 row in set (0.00 sec)
+```
+
+### Consumer
+
+Create table has been propagated to the consumer:
+
+```
+mysql> show tables;
++-----------------+
+| Tables_in_flbDb |
++-----------------+
+| pet             |
++-----------------+
+1 row in set (0.00 sec)
+```
+
+## Sample MYSQL Commands
+
+For additional mysql commands, checkout [MYSQL-COMMANDS](./MYSQL_COMMANDS.md)
```

Comment:
Other sample commands you want want to run

```bash
INSERT INTO pet VALUES ('Puffball','Diane','hamster','f','1999-03-30');
INSERT INTO pet VALUES ('Jack','Peter','dog','m','1999-03-30');
UPDATE pet SET birth = '1989-08-31' WHERE name = 'Jack';
ALTER TABLE pet ADD COLUMN death DATE;
DELETE from pet where name="Puffball";
INSERT INTO pet VALUES ("Spot", "Jane", "dog", "m", "2010-11-2", Null);
UPDATE pet SET death="2020-6-10" WHERE name="Spot";
```

For addition commands ... (I think we should move all Pet related stuff to this level.. then leave all other samples in the COMMANDS. Please update the README as well).